mockgen/mockgen.go

```diff
@@ -304,10 +304,16 @@ func (g *generator) Generate(pkg *model.Package, outputPkgName string, outputPac
 	}
 	sort.Strings(sortedPaths)
 
+	var importPaths []string
+	for _, pth := range sortedPaths {
```

Comment:
Looks like you don't need `importPaths` or the loop. You should just be able to pass in `sortedPaths` into `createPackageMap`

Reply:
done